webapp/packages/core-navigation-tree/src/NodesManager/isFolderNode.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import type { NavNode } from './EntityTypes.js';
+import { NAV_NODE_TYPE_FOLDER } from './NAV_NODE_TYPE_FOLDER.js';
+
+export function isFolderNode(node: NavNode | undefined): boolean {
```

Comment:
it's better to name it ConnectionFolder because we have different folders in nav tree (example: folders in datasource, folders in resources, folders with connections)